CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Changelog
 
+## TBD
+
+### Dependencies
+
+Update bugsnag-android to [v6.12.0](https//github.com/bugsnag/bugsnag-android/releases/tag/v6.12.0) [#2340](https://github.com/bugsnag/bugsnag-js/pull/2340)
+
```

Comment:
@Cawllec - this appears to have missed the current [Unreleased] section - probably where we changed format last year?

Reply:
Yeah, I'll make a ticket to add to that instead of TBD if it's present